lib/cdo/profanity_filter.rb

```diff
@@ -17,6 +37,19 @@ class ProfanityFilter
     fick: %w(sv) # "got" in Swedish
   }
 
+  # Look for profanities in a given text, return true if any expletive found
+  # or false if no profanities found. Checks against all locales.
+  #
+  # @param [String] text to check for profanity
+  # @return [String, nil] The first instance of profanity (if any) or nil (if none)
+  def self.find_potential_profanity_in_any_language(text)
+    WEB_PURIFY_SUPPORTED_LANGUAGES.each do |language_code|
```

Comment:
On https://www.webpurify.com/documentation/additional/language/ it says:
>For Enterprise Customers:
>You may specify multiple languages per request (ex: lang=en,es,it,fr) and the submitted text will be checked against each of these languages.
Do you know if we're an Enterprise customer, and if so, it might be possible to remove the loop here

Reply:
Just checked: we _do_ have an enterprise plan